api/envoy/config/overload/v3/overload.proto

```diff
@@ -136,12 +136,25 @@ message OverloadAction {
   // A set of triggers for this action. The state of the action is the maximum
   // state of all triggers, which can be scaling between 0 and 1 or saturated. Listeners
   // are notified when the overload action changes state.
+  // An overload manager action can only have one trigger for a given resource.
```

Comment:
It is not clear to me what defines a resource?
Is a "resource" identified by the [Trigger.name](https://github.com/envoyproxy/envoy/blob/9b3d7d595f899a216e55b85716963e1b8b726827/api/envoy/config/overload/v3/overload.proto#L71) field? If so, please add a reference to that field.